pkgs/applications/networking/twingate/default.nix

```diff
@@ -0,0 +1,51 @@
+{ autoPatchelfHook
+, curl
+, dpkg
+, dbus_daemon
+, fetchurl
+, lib
+, libnl
+, libudev
+, stdenv
+, pkgs
+}:
+
+stdenv.mkDerivation rec {
+  pname = "twingate";
+  version = "1.0.12";
+  
+  src = fetchurl {
+    url = "gs://twingate-prod-binaries/client/linux/DEB/1.0.12/twingate-amd64.deb";
```

Comment:
We should use the public endpoint and not the GCS endpoint. How does it even work? It is a `gs` scheme and not `https` 🤔